code/src/components/HappyForm.js

```diff
@@ -0,0 +1,48 @@
+import React, { useState } from "react"
+import "./HappyForm.css"
+
+export const HappyForm = (props) => {
```

Comment:
Maybe "happy" is not necessary?
```suggestion
export const Form = (props) => {
```